AsyncDisplayKitTests/ASSnapshotTestCase.m

```diff
@@ -37,8 +37,6 @@ @implementation ASSnapshotTestCase
 
 + (void)hackilySynchronouslyRecursivelyRenderNode:(ASDisplayNode *)node
 {
-  ASDisplayNodeAssertNotNil(node.calculatedLayout, @"Node %@ must be measured before it is rendered.", node);
```

Comment:
It does *not* passes in most of the tests at that point any test at this point. `layoutIfNeeded` is called in `recursivelyEnsureDisplaySynchronously` and there the layout becomes a `calculatedLayout` the `calculatedLayout`. Before the `calculatedLayout` is not existent.